src/spikeinterface/comparison/comparisontools.py

```diff
@@ -8,7 +8,7 @@
 import numpy as np
 
 
-def count_matching_events(times1, times2, delta=10):
+def count_matching_events(times1: list, times2: list, delta=10):
```

Comment:
Is list enforced here. We've been having this debate recently to find a better name, but most of the times these functions can take in:

```
list | np.ndarray | tuple
```
So unless this can only take a list I would probably do all three.